### PR TITLE
Allow specify dest sourcemaps option as string

### PR DESCRIPTION
**Pull request by TrySound**
I think this will be much prettier and consistently.

```js
gulp.src('path', { sourcemaps: true })
  .pipe(gulp.dest('path', { sourcemaps: '.' }));
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -226,7 +224,7 @@ vfs.dest('./', {
 });
 ```
 
-Type: `Boolean` or `Object`
+Type: `Boolean`, `String` or `Object`
```
Comment (contra) [inline README.md]: Needs more docs on what each type does

**Comment by TrySound** [comment]
@contra Done.

File: README.md [hunk 2]
```diff
@@ -199,7 +199,8 @@ Default: `true` (always overwrite existing files)
 
 ##### `options.sourcemaps`
 
-Enables sourcemap support on files passed through the stream.  Will write inline soucemaps if specified as `true`. Uses [gulp-sourcemaps] under the hood.
+Enables sourcemap support on files passed through the stream.  Will write inline soucemaps if specified as `true`.
+Will write sourcemaps file in relative to destination folder if specified as `string`. Uses [gulp-sourcemaps] under the hood.
```
Comment (contra) [inline README.md]: I think it would be clearer to say "Specifying a string is shorthand for the path option"

**Comment by TrySound** [comment]
Fixed.

**Comment by TrySound** [comment]
@contra Will you publish patch release?

**Comment by phated** [comment]
This requires a minor release

**Comment by TrySound** [comment]
@phated Oh, sure.

**Comment by phated** [comment]
I tagged this.  Will publish sometime after the CI tests run

**Comment by TrySound** [comment]
But tests are already passed in CI.

**Comment by phated** [comment]
Published as 2.4.0

**Comment by TrySound** [comment]
Nice. Thanks.

**Pull request merged**